python_ta/checkers/bad_loop_indexing_checker.py

```diff
@@ -0,0 +1,109 @@
+"""checker for bad indexing in a loop.
+"""
+
+import astroid
+from pylint.interfaces import IAstroidChecker
+from pylint.checkers import BaseChecker
+from pylint.checkers.utils import check_messages
+from typing import List, Optional
+from astroid.node_classes import NodeNG
+
+
+class BadLoopIndexingChecker(BaseChecker):
+    __implements__ = IAstroidChecker
+    # name is the same as file name but without _checker part
+    name = 'bad_loop_indexing'
+    # use dashes for connecting words in message symbol
+    msgs = {'E9994': (f'Bad use of loop variable %s in for loop',
```

Comment:
Put the quotation marks around the variable here (i.e., `...loop variable "%s" in for loop`) so that you can remove them below.